src/Avatar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { stringToColor } from '../util/colorMapping';
+import { Props } from './props';
+import { styles } from './styles';
+
+export const Avatar = ({
+  avatarUrl = undefined,
+  username = undefined,
+  isOnline = undefined,
+  showOnline = false,
+  style = {},
+  onClick = () => {}
+}: Props) => {
+
+  const text = username ? username.substring(0, 2).toUpperCase() : ''
+  const color = stringToColor(username)
+
+  return (
+    <div style={{ width: '48px', height: '48px' }} onClick={onClick}>
+      <div style={{ height: '0px' }}>
+        <div
+          className='ce-avatar'
+          style={{
+            ...styles.avatar,
+            ...style,
+            ...{
+              backgroundColor: avatarUrl ? '#FFFFFF' : color,
+              backgroundImage: avatarUrl && `url(${avatarUrl})`,
+            }
+          }}
+        >
+          <div
+            className='ce-avatar-text'
+            style={styles.avatarText}
+          >
+            {!avatarUrl && text}
+          </div>
+        </div>
+      </div>
+
+      {
+        showOnline &&
```

Comment:
Should we have two variables, showOnline and isOnline? Or make isOnline rendered if !== undefined.

I actually don't know lets discuss that a bit 🤔

Reply:
bit of a toss up since both ways accomplish the same thing;

but consider someone trying to use the component- they probably won't even realize you can toggle the online indicator by setting is to undefined. Some might even assume it would just default to false

so I think keeping them separate makes it much clearer as to how to use the component- best documentation is clear and unambiguous code!

Reply:
Fair enough - lets keep it 👍